SeoPro/resources/lang/es/fieldsets/sections.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+return [
+
+    'enabled' => 'Habilitado',
+    'enabled_instruct' => 'Deshabilitando esta sección la excluirá de reportes y del sitemap, y nada será renderizado a través de la etiqueta.',
+
+    'meta_section' => 'Metadatos',
+
+    'title' => 'Meta Título',
+    'title_instruct' => "Seleciona un campo existente para que ejerza como el meta título **por defecto** de esta sección.",
+
+    'description' => 'Meta Descripción',
+    'description_instruct' => "Seleciona un campo existente para que ejerza como la meta descripción **por defecto** de esta sección.",
```

Comment:
Is there a reason why these quotes are not single quotes? I just realised there are a few instances where double quotes are used instead of single.